src/federated_main.py

```diff
@@ -29,8 +29,8 @@
     args = args_parser()
     exp_details(args)
 
-    if args.gpu_id:
-        torch.cuda.set_device(args.gpu_id)
+    if args.gpu:
```

Comment:
Change readme commands to use --gpu=cuda:0 instead of --gpu=0 (since it might trip others up)

Reply:
Changed.